app/pages/analyst/cbc/[cbcId].tsx

```diff
@@ -267,6 +267,7 @@ const Cbc = ({
           rowId: query?.cbcByRowId?.cbcDataByCbcId?.edges[0].node.rowId || null,
           cbcDataPatch: {
             jsonData: {
+              ...query?.cbcByRowId?.cbcDataByCbcId?.edges[0].node.jsonData,
```

Comment:
I know this is a temp fix until we've included this data in `schemaUtils`, think we can add a comment making a note of that. Something like:

```
//TODO: Update when working on NDT-725
```